### PR TITLE
make tests compatible with django 1.8

### PR DESCRIPTION
**Pull request by nickpell**
@emord cc: @snopoke

### CODE REVIEW

File: auditcare/tests/auth.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 class authenticationTestCase(TestCase):
     def setUp(self):
+        super(authenticationTestCase, self).setUp()
```
Comment (emord) [inline auditcare/tests/auth.py]: Weird this class starts with a lowercase letter

Comment (nickpell) [inline auditcare/tests/auth.py]: it pained me a little to type that

Comment (emord) [inline auditcare/tests/auth.py]: is it hard to change?

Comment (nickpell) [inline auditcare/tests/auth.py]: probably not, I've just been being selective about what rabbit holes to go down

**Pull request merged**